ical/README.md

```diff
@@ -0,0 +1,46 @@
+# cljsjs/ical
+
+[](dependency)
+```clojure
+[cljsjs/ical "1.2.1-1"] ;; latest release
+```
+[](/dependency)
+
+This jar comes with `deps.cljs` as used by the [Foreign Libs][flibs]
+feature of the Clojurescript compiler. After adding the above
+dependency to your project you can require the packaged library like
+so:
+
+```clojure
+(ns application.core
+  (:require cljsjs.ical))
+```
+
+Here's a simple example of how to parse the contents of an `.ics` file:
+
+```
```

Comment:
Very minor style issue here: extra newline and missing clojure filetype from code block.

Looks good otherwise.